### PR TITLE
Added miscellaneous missing videos.

### PR DESCRIPTION
**Pull request by davidstap**
Added a remainder of miscellaneous missing videos:

        modified:   data/xml/2014.lilt.xml
        modified:   data/xml/2020.cl.xml
        modified:   data/xml/2020.tacl.xml
        modified:   data/xml/2021.cl.xml
        modified:   data/xml/2021.eacl.xml
        modified:   data/xml/2021.humeval.xml
        modified:   data/xml/2021.tacl.xml
        modified:   data/xml/2022.acl.xml
        modified:   data/xml/2022.cl.xml
        modified:   data/xml/2022.dialdoc.xml
        modified:   data/xml/2022.tacl.xml
        modified:   data/xml/P14.xml
        modified:   data/xml/Q13.xml
        modified:   data/xml/Q14.xml

### CODE REVIEW

File: data/xml/Q13.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
       <url hash="84ba73ea">Q13-1002</url>
       <bibkey>pitler-etal-2013-finding</bibkey>
       <video href="Q13-1002.mp4"/>
+      <video href="Q13-1002.mp4"/>
     </paper>
```
Comment (mjpost) [inline data/xml/Q13.xml:40]: These in Q13 are all duplicates

File: data/xml/2014.lilt.xml [hunk 2]
```diff
@@ -13,6 +13,7 @@
       <author><first>Valeria</first><last>de Paiva</last></author>
       <url hash="d6aae7a0">2014.lilt-9.1</url>
       <bibkey>zaenen-etal-2014-introduction</bibkey>
+      <video href="2014.lilt-9.1.mp4"/>
```
Comment (mjpost) [inline data/xml/2014.lilt.xml:16]: This one is very off (it's NAACL 2022)

**Review by mjpost — COMMENTED** [review]
How are you matching these, and where are these old videos coming from?

File: data/xml/2014.lilt.xml (hunk 2 above)
Comment (davidstap) [reply data/xml/2014.lilt.xml]: It is! Due to an issue with title matching in my video script. Removed it now.

**Comment by davidstap** [comment]
I removed duplicate videos and a wrong match. The remaining videos should match (I spot checked a few).

File: data/xml/Q13.xml (hunk 1 above)
Comment (davidstap) [reply data/xml/Q13.xml]: fixed it.

**Review by mjpost — APPROVED** [review]
I spot-checked a handful and they seem to be good.

**Pull request merged**